bee-message/src/payload/receipt/migrated_funds_entry.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    constants::IOTA_SUPPLY, output::SignatureLockedSingleOutput, payload::receipt::TailTransactionHash, Error,
+};
+
+use bee_common::packable::{Packable, Read, Write};
+
+use core::ops::RangeInclusive;
+
+/// Range of allowed amount for migrated funds entries.
+pub const MIGRATED_FUNDS_ENTRY_AMOUNT: RangeInclusive<u64> = 1_000_000..=IOTA_SUPPLY;
```

Comment:
This might be better named `MIGRATED_FUNDS_VALID_ENTRY_AMOUNT` or `VALID_MIGRATED_FUNDS_ENTRY_AMOUNT` (probably too picky though).